alacritty/src/renderer/mod.rs

```diff
@@ -1007,16 +1015,22 @@ impl<'a, C> RenderApi<'a, C> {
 
     #[inline]
     fn add_render_item(&mut self, cell: RenderableCell, glyph: &Glyph) {
+        let layer_id =
+            if glyph.colored { LayerType::ColoredGlyphs } else { LayerType::RegularGlyphs };
```

Comment:
```suggestion
        let layer_type =
            if glyph.colored { LayerType::ColoredGlyphs } else { LayerType::RegularGlyphs };
```

Seems like layer_type would be the correct name here? Since it's not an ID?